tasks/managed/push-snapshot/tests/mocks.sh

```diff
@@ -49,8 +50,8 @@ function skopeo() {
 }
 
 function get-image-architectures() {
-    echo '{"platform":{"architecture": "amd64", "os": "linux"}, "digest": "abcdefg"}'
-    echo '{"platform":{"architecture": "ppc64le", "os": "linux"}, "digest": "deadbeef"}'
+  echo '{"platform":{"architecture": "amd64", "os": "linux"}, "digest": "abcdefg"}'
```

Comment:
Can we reverse the order of the architecture to do some test? Seems "amd64" is not always the first one being returned from the user's case.